### ISSUE TITLE
Training loss in training step1&2

### ISSUE THREAD

**Issue by zhangpur** [huang-xx/STGAT#15]
Hi! Thanks for sharing your work. 
I don't understand the design that you use observe data and groudtruth to train lstm in traing_step1&2, is it like an autoencoder?  By the way, the loss mark is at the predict phase.
Thanks for your help.
